executor/newdeploy/newdeploymgr.go

```diff
@@ -548,9 +548,7 @@ func (deploy *NewDeploy) fnDelete(fn *crd.Function) (*fscache.FuncSvc, error) {
 }
 
 func (deploy *NewDeploy) getObjName(fn *crd.Function) string {
-	return fmt.Sprintf("%v-%v",
-		fn.Metadata.Name,
-		deploy.instanceID)
+	return strings.ToLower(fmt.Sprintf("newdeploy-%v-%v", strings.ToLower(fn.Metadata.Name), deploy.instanceID))
```

Comment:
Why add "newdeploy"?

Reply:
[Resolved because I noticed you added it for poolmgr as well, sounds good to me.]